pkg/commands/secretstore/flags.go

```diff
@@ -0,0 +1,71 @@
+package secretstore
+
+import (
+	"github.com/fastly/cli/pkg/cmd"
+	"github.com/fastly/kingpin"
+)
+
+func storeIDFlag(dst *string) cmd.StringFlagOpts {
+	return cmd.StringFlagOpts{
+		Name:        "store-id",
+		Short:       's',
+		Description: "Store ID",
+		Dst:         dst,
+		Required:    true,
+	}
+}
+
+func storeNameFlag(dst *string) cmd.StringFlagOpts {
+	return cmd.StringFlagOpts{
+		Name:        "name",
+		Short:       'n',
+		Description: "Store name",
+		Dst:         dst,
+		Required:    true,
+	}
+}
+
+func secretNameFlag(dst *string) cmd.StringFlagOpts {
+	return cmd.StringFlagOpts{
+		Name:        "name",
+		Short:       'n',
+		Description: "Secret name",
+		Dst:         dst,
+		Required:    true,
+	}
+}
+
+func secretFileFlag(dst *string) cmd.StringFlagOpts {
+	return cmd.StringFlagOpts{
+		Name:        "file",
+		Short:       'f',
+		Description: "Read secret value from file instead of prompt",
+		Dst:         dst,
+		Required:    false,
+	}
+}
+
+func secretStdinFlag(dst *bool) cmd.BoolFlagOpts {
+	return cmd.BoolFlagOpts{
+		Name:        "stdin",
+		Description: "Read secret value from STDIN instead of prompt",
+		Dst:         dst,
+		Required:    false,
+	}
+}
+
+func cursorFlag(dst *string) cmd.StringFlagOpts {
+	return cmd.StringFlagOpts{
+		Name:        "cursor",
+		Short:       'c',
+		Description: "Pagination cursor (Use 'next_cursor' value from list output)",
+		Dst:         dst,
+	}
+}
+
+func limitFlag(cmd *kingpin.CmdClause, dst *int) {
```

Comment:
I think we should move this to the same package as [`RegisterFlag`](https://github.com/fastly/cli/blob/be34d9abbb982a91c03c66f40ed996d7cf33f2d0/pkg/cmd/flags.go#L38-L50) (which also should really be named `RegisterFlagString`) and name it `RegisterFlagInt`.

It makes the code consistent and a new user won't have to guess why there is a difference, as the calling code is currently...

```go
c.RegisterFlag(...)
c.RegisterFlagBool(...)
limitFlag(...)
```

...so it kinda stands out compared to something like...

```go
c.RegisterFlagString(...)
c.RegisterFlagBool(...)
c.RegisterFlagInt(...)
```

Reply:
Great idea. Done in ab89227.